### PR TITLE
🌐(courses) translate course run language

### PR DESCRIPTION
**Pull request by 2018D**
## Purpose

Translate languages used for course run through multiselect field.
They were untranslated up till now.

## Proposal

Make the values used in setting translatable.
Fix the multiselect field to return lower string values only after translation is done.

### CODE REVIEW

File: src/richie/apps/core/fields/multiselect.py [hunk 1]
```diff
@@ -23,10 +23,12 @@ def to_sentence(elements):
...
-        return _("{:s} and {:s}").format(
-            ", ".join(map(str.lower, elements[:-1])), str.lower(elements[-1])
-        )
-    return _(" and ").join(map(str.lower, elements))
+        return (
+            _("{:s} and {:s}").format(
+                ", ".join(map(str, elements[:-1])), str(elements[-1])
+            )
+        ).lower()
+    return (_(" and ").join(map(str, elements))).lower()
```
Comment (sampaccoud) [inline src/richie/apps/core/fields/multiselect.py]: can you add tests that validate the behavior you are targetting?

Comment (2018D) [reply src/richie/apps/core/fields/multiselect.py]: This ensures that the following tests succeed in :

tests/apps/courses/test_models_course_run.py
- test_models_course_run_get_languages_display_one_language
- test_models_course_run_get_languages_display_request
- test_models_course_run_get_languages_display_three_languages                                                                                 
- test_models_course_run_get_languages_display_two_languages

in tests/apps/courses/test_templates_course_detail.py
- test_templates_course_detail_cms_draft_content
- test_templates_course_detail_cms_published_content
- test_templates_course_detail_state_with_cta
- test_templates_course_detail_state_without_cta

in tests/apps/courses/test_templates_course_run_detail.py
test_templates_course_run_detail_cms_draft_content
test_templates_course_run_detail_cms_published_content
test_templates_course_run_detail_no_index
test_templates_course_run_detail_state_with_cta
test_templates_course_run_detail_state_without_cta

Comment (2018D) [reply src/richie/apps/core/fields/multiselect.py]: Here's the error log without moving the lower to later:
```
File "/app/src/richie/apps/core/fields/multiselect.py" in func
  259.                         [choicedict.get(value, value) for value in getattr(self, name)]

File "/app/src/richie/apps/core/fields/multiselect.py" in to_sentence
  29.     return _(" and ").join(map(str.lower, elements))

File "/usr/local/lib/python3.6/site-packages/django/utils/functional.py" in __wrapper__
  109.                 return getattr(res, method_name)(*args, **kw)

Exception Type: TypeError at /en/courses/new-course/
Exception Value: descriptor 'lower' requires a 'str' object but received a '__proxy__'
```

Comment (sampaccoud) [reply src/richie/apps/core/fields/multiselect.py]: Ok sorry. I thought you changed the behavior. This should be explained in your commit message maybe.

**Comment by sampaccoud** [comment]
:tada: great! Sorry again for having been slow to understand :blush:

**Pull request merged**